server/routes/friendRouter.js

```diff
@@ -0,0 +1,53 @@
+const router = require('express').Router();
+const User = require('../models/userModel');
+
+router.get('/usersList', async (req, res) => {
+    try {
+        const user = await User.find({});
+        res.json({
+            user: user,
+        });
+    } catch (err) {
+        res.status(500).json({ msg: err.message });
+    }
+});
+
+router.put('/follow', async (req, res) => {
```

Comment:
We need to start gating a bunch of these endpoints, so we can introduce a middleware that authenticates the user coming in. When we do that, you probably don't need to pass in `currentUser` any more